### PR TITLE
add badges on readme file

### PR DESCRIPTION
**Pull request by arthru**
As suggested by @bwrsandman, here are the badges !

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Hi @arthru,

You should do the README with extension and format .md, that it's the convention across OCA. You can see an example here:

https://raw.githubusercontent.com/OCA/account-invoicing/8.0/README.md

**Comment by bwrsandman** [comment]
Indeed... .md format is used uniformly by OCA

**Comment by arthru** [comment]
Updated to markdown...

**Comment by pedrobaeza** [comment]
Thanks!
:+1:

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[![Build Status](https://travis-ci.org/OCA/connector-prestashop.svg?branch=8.0)](https://travis-ci.org/OCA/connector-prestashop)
```
Comment (bwrsandman) [inline README.md]: Mistake here. Should be: `branch=7.0`

**Comment by arthru** [comment]
Just fixed the wrong version !

**Comment by bwrsandman** [comment]
LGTM :+1:

**Pull request merged**